addons/web/static/src/views/list/list_renderer.js

```diff
@@ -261,7 +261,7 @@ export class ListRenderer extends Component {
         table.classList.add("o_list_computing_widths");
 
         const headers = [...table.querySelectorAll("thead th")];
-        const columnWidths = headers.map((th) => th.offsetWidth);
+        const columnWidths = headers.map((th) => th.getBoundingClientRect().width);
```

Comment:
Seems fine but I'm curious what problem it solves?

Reply:
To be short: rounding issue.

Actually, offsetWidth returns an integer value, rounded by the browser, which can lead to having the sum of the element's widths being 1px bigger than their container... and this is the case in this PR as the checkbox width is slightly different.

Using getBoundingClientRect returns a float and allows to have a more precise calculation.

Reply:
I wonder if we may get situations where the problem persists with this version, maybe we should round this value up so that the width needed for the column is always overestimated? Fractional pixel problems are always hard to test unfortunately. If this solves a problem in practice it's fine with me :)

Reply:
We can manually always round it up or down: the issue is that with offsetWidth, this is the browser who decides...

I'll try anyway and see what will happen. Thanks for the suggestion 🙂

Reply:
Update: actually rounding *up* doesn't works (or at least breaks all the time :nerd_face: ) ; rounding *down* works though. Actually it makes more sense as we try to distribute as much space as possible, but never give more than the available space.